projects/opennavsurf-bag/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: "https://github.com/OpenNavigationSurface/BAG"
+language: c++
+primary_contact: "selimnairb@gmail.com"
+auto_ccs:
+  - "bmiles@ccom.unh.edu"
+  - "brc@ccom.unh.edu"
+  - "glen.rice@noaa.gov"
+sanitizers:
+  - address
+  - memory:
+     experimental: True
```

Comment:
You should probably remove the experimental part.

Reply:
Thanks, I'll clean this up as I go through the setup process.